ycmd/tests/subcommands_test.py

```diff
@@ -24,17 +24,20 @@
                           PathToTestFile,
```

Comment:
Yes. I thought about doing this, but wasn't sure of your appetite for the organisation. Proper organisation of tests is essential to avoid duplication, etc., though the correct organisation is always debatable. 

I suspect, for the subcommands at least, we should organise them around completer, then subcommand, e.g.

```
tests/subcommands_test_clang.py:
   class ClangSubcommandTest_GetType:
       ...
   class ...

tests/subcommands_test_jedi.py:
   ...
```

Up for debate ;)